isthmus/src/main/java/io/substrait/isthmus/IsthmusEntryPoint.java

```diff
@@ -61,20 +68,40 @@ enum OutputFormat {
   // this example implements Callable, so parsing, error handling and handling user
   // requests for usage help or version help can be done with one line of code.
   public static void main(String... args) {
-    int exitCode = new CommandLine(new PlanEntryPoint()).execute(args);
+    logger.debug(Arrays.toString(args));
```

Comment:
do we need this?

Reply:
Just deleted log options, and added https://github.com/remkop/picocli/blob/main/picocli-codegen/README.adoc#213-gradle to helps us with `--help` and `--version` options.